win32.go

```diff
@@ -0,0 +1,121 @@
+// +build windows
+
+package main
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+	"syscall"
+	"unsafe"
+
+	"github.com/mitchellh/go-ps"
```

Comment:
We removed this dependency in favor of [gopsutil](https://github.com/shirou/gopsutil). Can you double check if we can also use `gopsutil` here?

Reply:
I'm checking there's a difference in the functon that returns the processes.  
go-ps executes only one call to CreateToolhelp32Snapshot and then will loop through the snapshot with Process32First and Process32Next, which is very fast.
gopsutil executes a call to get the process pids, but when we acces the process name, it executes a call to  CreateToolhelp32Snapshot  for each process, which is very slow.
I'm still investigating there's a way to have the same behaviour with gopsutil.

Reply:
That would be a valid reason to use `go-ps` in that case.

Reply:
dependency removed and only needed code added